stablehlo/dialect/StablehloOps.cpp

```diff
@@ -857,12 +857,7 @@ LogicalResult ConvolutionOp::verify() {
 
 void ConvertOp::build(OpBuilder& builder, OperationState& result, Value operand,
                       Type resultElementTy) {
-  Type resultTy;
-  Type operandTy = operand.getType();
-  if (auto rankedTy = operandTy.dyn_cast<RankedTensorType>())
-    resultTy = RankedTensorType::get(rankedTy.getShape(), resultElementTy);
-  else
-    resultTy = UnrankedTensorType::get(resultElementTy);
+  auto resultTy = operand.getType().cast<TensorType>().clone(resultElementTy);
```

Comment:
Would you mind holding off on this change? I'm actually about to change this code to remove unranked dynamism support.

Reply:
Sure. Let me know once your changes are done.

Reply:
Should be done soon, but I can't provide a reliable estimate (ideally today, otherwise should be next week). WDYT about undoing these changes and committing the rest? I don't want to block you.

Reply:
make sense. updone!